src/utils/shim.js

```diff
@@ -1,21 +1,3 @@
-/*
```

Comment:
@alazarevski revert this removal, it's important to have this `copyright notice` besides this #LGTM